src/rt/typeinfo/ti_long.d

```diff
@@ -28,7 +26,7 @@ class TypeInfo_l : TypeInfo
 
     override size_t getHash(scope const void* p)
     {
-        return rt.util.hash.hashOf(p[0 .. long.sizeof], 0);
+        return hashOf(*cast(long*)p, 0);
```

Comment:
No need for `0` second argument. Currently it will be inferred, and in the future an explicit chaining argument might result in different behavior (https://github.com/dlang/druntime/pull/2246).

Reply:
done